server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/bitcoin/sources/bittrex/Bittrex.java

```diff
@@ -0,0 +1,19 @@
+package com.generalbytes.batm.server.extensions.extra.bitcoin.sources.bittrex;
+
+import com.generalbytes.batm.server.extensions.extra.bitcoin.sources.bittrex.dto.BittrexTickerDto;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+@Produces(MediaType.APPLICATION_JSON)
+@Path("/v3")
+public interface Bittrex {
+
+    @GET
+    @Path("/markets/{marketSymbol}/ticker")
+    BittrexTickerDto getTicker(@PathParam("marketSymbol") String marketSymbol);
+
+}
```

Comment:
why was the previous implementation using the XChange library (the getExchange method) not sufficient? This is also fine if it works

Reply:
I just thought this will be an easier and more readable solution.

Reply:
Outdated, previous implementation without changes